worker/sshserver/server.go

```diff
@@ -92,7 +99,7 @@ func NewServerWorker(config ServerWorkerConfig) (worker.Worker, error) {
 		s.config.Listener = listener
 	}
 
-	listener, closeAllowed := NewSSHServerListener(s.config.Listener)
```

Comment:
We don't need closeAllowed for production code, as it's possible that an accept might never be called.

Reply:
true, this is a test only issue.